src/decoders/image.rs

```diff
@@ -31,6 +31,11 @@ pub struct RawImage {
   pub cfa: CFA,
   /// how much to crop the image to get all the usable area, order is top, right, bottom, left
   pub crops: [usize;4],
+
+  /// Area of the sensor that is masked to prevent it from receiving light. Used to calculate
+  /// black levels and noise. Order is horizontal left, size; vertical top, size
+  pub blackarea: Option<[(usize,usize);2]>,
```

Comment:
This is a slightly strange API. Wouldn't it make more sense to just expose `blackareah` and `blackareav`? Maybe something like:

```rust
blackareah: Option<(usize,usize)>,
blackareav: Option<(usize,usize)>,
```

Those are simplifications as they don't allow multiple vertical and horizontal areas but in practice those are not really needed in current models. But maybe to be future proof:

```rust
blackareah: Vec<(u64,u64)>,
blackareav: Vec<(u64,u64)>,
```

Or even something that unifies everything:

```rust
blackareas: Vec<(u64,u64,u64,u64)>,
```

Reply:
Originally I was trying to minimize the api surface, but I think the `blackareas: Vec<(u64,u64,u64,u64)>` suggestion is a good compromise.

Reply:
I simplified the original API just so I could write the TOML files in an easier way. But that's just an internal interface. External APIs need to be done properly. I'd like to do 1.0 soon and wouldn't want to do breaking changes after that unless really needed.

Reply:
Updated to `Vec<(u64,u64,u64,u64)>`